Makefile.am

```diff
@@ -1,3 +1,5 @@
 SUBDIRS = lib
 ACLOCAL_AMFLAGS = -I m4
 AM_LDFLAGS=-no-undefined
+pkgconfigdir = $(libdir)/pkgconfig
```

Comment:
If you call `PKG_INSTALLDIR` from pkg.m4(7) in configure.ac, it will define this directory for you (with the same default value) but it will also add a `--with-pkgconfigdir` to ./configure

Reply:
fixed